README.md

```diff
@@ -80,22 +88,22 @@ The path to your node webkit app. It supports [simple-glob](https://github.com/j
 
 
 #### options.version
-Type: `String`
-Default value: `'latest'`
+Type: `String`  
```

Comment:
No, the two spaces indicate a line break in markdown. They were missing from some options which made the Type and Default value show on the same line. The two spaces before an *empty line* don't matter but I added them as opposed to removed them for consistency because I think there were more that had them than didn't.